packages/core/src/geometry/Buffer.js

```diff
@@ -47,6 +47,7 @@ export default class Buffer
     // TODO could explore flagging only a partial upload?
     /**
      * flags this buffer as requiring an upload to the GPU
+     * @param {(ArrayBuffer| SharedArrayBuffer|ArrayBufferView)=} data the data to update in the buffer.
```

Comment:
What's up with this `{(` ... `)=}` syntax? Is that a thing or a typo?

Reply:
I wanted to make `data` optional. Thought I'm not 100% familiar with JSDoc syntax.

This results in this Typescript definition:
`update(data?: ArrayBuffer | SharedArrayBuffer | ArrayBufferView): void;`

Reply:
The brackets `()` might not be needed

I used this as reference:
http://usejsdoc.org/tags-param.html#optional-parameters-and-default-values

Reply:
Ahh okay. Then it should look like this:
```
@param {ArrayBuffer|SharedArrayBuffer|ArrayBufferView} [data]
```

Reply:
Cheers! Updated